src/Infrastructure/Infrastructure.csproj

```diff
@@ -8,6 +8,9 @@
 
   <ItemGroup>
     <PackageReference Include="CsvHelper" Version="15.0.10" />
+    <PackageReference Include="EasyCache.MemCache" Version="1.0.0" />
+    <PackageReference Include="EasyCache.Memory" Version="1.0.0" />
+    <PackageReference Include="EasyCache.Redis" Version="1.0.0" />
```

Comment:
Is adding all dependencies good soluton? Even rest of them aren't used.

Reply:
It needs to be an example for all providers. I took as an example the method of implementing InMemory and SQL Server in CleanArchitecture. @enisn